### PR TITLE
enhancement:  manifest validations

### PR DESCRIPTION
**Pull request by Hardeep8587**
## Change Description

Shipper/consignee pop ups ,
 city state country pop ups,
ACE date,
main driver/co driver

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Hardeep8587** [comment]
@kunalfleethawks Please review and merge! thanku !

File: src/app/pages/dispatch/cross-border/ace-documents/new-ace-manifest/new-ace-manifest.component.ts [hunk 1]
```diff
@@ -355,17 +304,17 @@ getThirdPartyStates(s,p){
...
-  this.apiService.getData('shippers').subscribe((result:any)=> {
-    this.shippers = result.Items;
-  });
   }
-  fetchConsignees(){
-    this.apiService.getData('receivers').subscribe((result:any)=> {
-      this.consignees = result.Items;
-    });
-    }
+  // fetchShippers(){
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/cross-border/ace-documents/new-ace-manifest/new-ace-manifest.component.ts:308]: If this code is not required, please remove

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.ts [hunk 2]
```diff
@@ -309,12 +246,12 @@ export class NewAciManifestComponent implements OnInit {
         this.acceptanceCities = result.Items;
       });
   }
-  getDocStates(){
-    this.apiService.getData('states')
-    .subscribe((result: any) => {
-      this.passengerDocStates = result.Items;
-    });
-  }
+  // getDocStates(){
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.ts:249]: If this code is not required please remove it.

File: src/app/pages/dispatch/dispatch.module.ts [hunk 3]
```diff
@@ -106,21 +106,36 @@ export class CustomAdapter extends NgbDateAdapter<string> {
...
-    if (value) {
-      let date = value.split(this.DELIMITER);
-      return {
-        year : parseInt(date[2], 10),
-        month : parseInt(date[1], 10),
-        day : parseInt(date[0], 10)
-      };
-    }
-    return null;
+  // fromModel(value: string | null): NgbDateStruct | null {
```
Comment (kunalfleethawks) [inline src/app/pages/dispatch/dispatch.module.ts]: Please remove this

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Please address the review comments

File: src/app/pages/dispatch/cross-border/ace-documents/new-ace-manifest/new-ace-manifest.component.ts (hunk 1 above)
Comment (Hardeep8587) [reply src/app/pages/dispatch/cross-border/ace-documents/new-ace-manifest/new-ace-manifest.component.ts:308]: please hold for a day, after integration will remove..

File: src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.ts (hunk 2 above)
Comment (Hardeep8587) [reply src/app/pages/dispatch/cross-border/aci-documents/new-aci-manifest/new-aci-manifest.component.ts:249]: please hold for a day, after integration will remove..

**Comment by Hardeep8587** [comment]
@kunalfleethawks please merge! I will remove all commented code, as soon as I get response...just keeping as security

**Pull request closed without merging**